Gemfile

```diff
@@ -10,12 +10,13 @@ gem 'dotenv-rails'
 gem 'enum_help'
 gem 'figaro'
 gem 'foreman', require: false
-gem 'hashie'
+gem 'hashie', '~> 3.5', '>= 3.5.5'
```

Comment:
yeah, i've never seen that either. Thats what you get when you copy from rubygems.org